tinuous

```diff
@@ -35,6 +35,20 @@ from yaml import safe_load
 
 log = logging.getLogger("tinuous")
 
+COMMON_STATUS_MAP = {
+    "success": "success",
+    "passed": "success",
+    "failure": "failed",
+    "failed": "failed",
+    "errored": "errored",
+    "timed_out": "errored",
+    "neutral": "incomplete",
+    "action_required": "incomplete",
+    "cancelled": "incomplete",
+    "skipped": "incomplete",
+    "stale": "incomplete",
+    # Error on unknown so we're forced to categorize them.
+}
```

Comment:
if coded this way, let's add an explicit safeguard against typos in the mapped into values, smth like this:

```suggestion
}


assert set(COMMON_STATUS_MAP.values()) == {'success', 'failed', 'errored', 'incomplete'}
```

Reply:
Done.